packages/devextreme-angular/README.md

```diff
@@ -1,26 +1,27 @@
 
-This project allows you to use [DevExtreme widgets](http://js.devexpress.com/Demos/WidgetsGallery/) in [Angular](https://angular.io/) applications.
+* [Documentation](https://js.devexpress.com/Angular/Documentation/Guide/Angular_Components/DevExtreme_Angular_Components/)
+* [Component Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Angular/Light/)
+* **Template Gallery**
+    * [Fluent theme](https://devexpress.github.io/devextreme-ui-template-gallery/fluent.blue.light/angular/default/#/crm-contact-list)
+    * [Generic theme](https://devexpress.github.io/devextreme-ui-template-gallery/generic.light.compact/angular/default/#/crm-contact-list)
+* Application Demos
+    * [Sales and Revenue Dashboard](https://github.com/DevExpress/SalesViewer)
+    * [Golf Course Booking Page](https://github.com/DevExpress/golfclub)
+    * [Interactive Election Results Map](https://github.com/DevExpress/dx-election)
 
-* [Documentation](https://js.devexpress.com/Documentation/Guide/Angular_Components/DevExtreme_Angular_Components/)
-* [Technical Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Angular/Light/)
-* Real-World Application Demos
-    * [Sales Viewer](https://github.com/DevExpress/SalesViewer)
-    * [Golf Club](https://github.com/DevExpress/golfclub)
-    * [DX Election](https://github.com/DevExpress/dx-election)
-* [API Reference](http://js.devexpress.com/Documentation/ApiReference/)
+## License
 
-## License ##
+DevExpress distributes the `devextreme-angular` package under the free and open-source **MIT** license.
 
-**DevExtreme Angular UI Components are released as a MIT-licensed (free and open-source) add-on to DevExtreme.**
+For more information on licensing requirements, read the [FAQ](https://js.devexpress.com/Licensing/). DevExpress offers a [free trial](http://js.devexpress.com/Buy/) of the product.
 
-Familiarize yourself with the [DevExtreme License](https://js.devexpress.com/Licensing/). [Free trial is available!](http://js.devexpress.com/Buy/)
+## Support & Feedback
 
-## Support & Feedback ##
+For more information on the Angular framework and how to use it, see the [official documentation](https://angular.io/docs).
 
-If you have questions regarding Angular functionality, consult [Angular docs](https://angular.io/docs).
-
-If you want to report a bug, request a feature, or ask a question, submit an [issue](https://github.com/DevExpress/devextreme-angular/issues) to this repo. Alternatively, you can contact us at the [DevExpress Support Center](https://www.devexpress.com/Support/Center) if you own an active DevExtreme license.
+If you own an active [commercial DevExtreme license](https://js.devexpress.com/Licensing/#Commercial), submit your questions, suggestions, and bug reports to the [DevExpress Support Center](https://www.devexpress.com/sc). Alternatively, create a new [issue](https://github.com/DevExpress/devextreme/issues) in the GitHub repo.
```

Comment:
I think we can omit 'commercial' here also (see above)

We should now remove a sentence about creation a new issue because it's impossible now.